Makefile

```diff
@@ -53,22 +53,18 @@ APPLICATION_VERSION_REDHAT = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
 # Release type
 # ---------------------------------------------------------------------
 
-# Add the current commit to the version if release type is "snapshot"
-RELEASE_TYPE ?= snapshot
 PACKAGE_JSON_VERSION = $(shell jq -r '.version' package.json)
-ifeq ($(RELEASE_TYPE),production)
+
+ifdef STABLE
```

Comment:
Just a minor niggle, but `STABLE` somehow feels like too short of a word for this (IYSWIM). What about something like `STABLE_RELEASE` ?